django/contrib/auth/management/commands/createsuperuser.py

```diff
@@ -20,6 +20,7 @@ class NotRunningInTTYException(Exception):
 class Command(BaseCommand):
     help = 'Used to create a superuser.'
     requires_migrations_checks = True
+    private_options = ('username', )
```

Comment:
In case `interactive==False`, required fields are retrieved from options.
`django.core.management.base.CommandError: You must use --username with --noinput.`